model/src/main/kotlin/AnalyzerRun.kt

```diff
@@ -41,4 +41,18 @@ data class AnalyzerRun(
     val environment: Environment,
     val config: AnalyzerConfiguration,
     val result: AnalyzerResult
-)
+) {
+    companion object {
+        /**
+         * A constant for an [AnalyzerResult] where all properties are empty.
+         */
+        @JvmField
+        val EMPTY = AnalyzerRun(
+            startTime = Instant.MIN,
+            endTime = Instant.MIN.plusSeconds(1),
```

Comment:
In the past, we used `Instant.EPOCH` as "dummy" start / end times. Shouldn't we stick to that convention here, too? (I believe it's fine to set start and end time to the same value here.)